charts/airflow-operator/values.yaml

```diff
@@ -0,0 +1,78 @@
+crd:
+  create: false
+certManager:
+  enabled: false
+
+runtimeVersions:
+  configMapName: ""
+  versionsJson: ""
+
+securityContext:
+  runAsNonRoot: true
+
+terminationGracePeriodSeconds: 10
+webhooks:
+  useCustomTlsCerts: false
+  certSecretName: "webhook-server-cert"
+  customCertsSecretName: "webhooks-tls-certs"
+  # for each of the following properties, the full certificate/key/CABundle should be provided in the following way:
+  # caBundle: |
+  #  -----BEGIN CERTIFICATE-----
+  #  .......................
+  #  -----END CERTIFICATE-----
+  caBundle: ""
+  tlsCert: ""
+  tlsKey: ""
+manager:
+  replicas: 1
+  metrics:
+    useSecuredEndpoint: false
+    image:
+      repository: gcr.io/kubebuilder/kube-rbac-proxy
```

Comment:
This image needs to go into ap-vendor and we need to update this to use the ap-vendor image that is under our control, not this third party controlled image.

Reply:
Also this is not showing up in `make show-docker-images`, so we need to fix that.